yt_dlp/extractor/bundesliga.py

```diff
@@ -0,0 +1,26 @@
+from .common import InfoExtractor
+from .jwplatform import JWPlatformIE
+
+
+class BundesligaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bundesliga\.com/[a-z]{2}/bundesliga/videos\?vid=(?P<id>\w+)'
```

Comment:
minor nitpick, but since I assume this is a JWPlatform id, might as well keep it as the same as JWPlatform id regex:

```suggestion
    _VALID_URL = r'https?://(?:www\.)?bundesliga\.com/[a-z]{2}/bundesliga/videos\?vid=(?P<id>[a-zA-Z0-9]{8})'
```